app/assets/stylesheets/scaffolds.css

```
@@ -0,0 +1,321 @@
+.button_to input[type="submit"]:hover {
+  background-color: var(--global-secondary-bgc-hover);
+}
+
+/****************
+page-nav
+****************/
+
+.page-nav {
+  --nav-fz: var(--global-fz-md);
+  --nav-mt: 24px;
+  font-size: var(--nav-fz);
```

Stylelint (17.2.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)